Index: WeBBAKI/target/classes/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.datasource.url=jdbc:mariadb://172.17.0.55:3306/NewWebbaki\r\nspring.datasource.driver-class-name=org.mariadb.jdbc.Driver\r\nspring.datasource.username=webbakiDev\r\nspring.datasource.password=Web%Ba!KI_2022\r\n\r\n\r\n# RECREATE DB\r\nspring.jpa.hibernate.ddl-auto=update\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/WeBBAKI/target/classes/application.properties b/WeBBAKI/target/classes/application.properties
--- a/WeBBAKI/target/classes/application.properties	
+++ b/WeBBAKI/target/classes/application.properties	
@@ -3,7 +3,6 @@
 spring.datasource.username=webbakiDev
 spring.datasource.password=Web%Ba!KI_2022
 
-
 # RECREATE DB
 spring.jpa.hibernate.ddl-auto=update
 
