Index: WeBBAKI/src/main/java/de/thb/webbaki/controller/ReportController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WeBBAKI/src/main/java/de/thb/webbaki/controller/ReportController.java b/WeBBAKI/src/main/java/de/thb/webbaki/controller/ReportController.java
--- a/WeBBAKI/src/main/java/de/thb/webbaki/controller/ReportController.java	
+++ b/WeBBAKI/src/main/java/de/thb/webbaki/controller/ReportController.java	
@@ -52,7 +52,7 @@
                                       Authentication authentication) {
 
         if (userService.getUserByEmail(authentication.getName()) != null){
-            User user = userService.getUserByEmail(authentication.getName());
+            User user = userService.getUserByUsername(authentication.getName());
             questionnaireFormModel.setUser(user);
             questionnaireService.saveQuestionaire(questionnaireFormModel);
         }
